README.md

```diff
@@ -25,6 +25,7 @@ This repository contains all of the code that runs on [worldcubeassociation.org]
   ```shell
   (cd WcaOnRails; bundle install && bin/yarn && bundle exec overcommit --install)
   ```
+  If some changes are made to this hook, you will have to update it using: `BUNDLE_GEMFILE=WcaOnRails/Gemfile bundle exec overcommit --sign`.
```

Comment:
Thanks for adding this! nit: Could you update this message to make it clear which director you're supposed to be in when you run it?

Reply:
Sure, done :)